fiftyone/operators/delegated.py

```diff
@@ -106,6 +110,18 @@ def set_pinned(self, doc_id, pinned=True):
         """
         return self._repo.set_pinned(_id=doc_id, pinned=pinned)
 
+    def set_label(self, doc_id, label):
+        """Sets the pinned flag for the given delegated operation.
+
+        Args:
+            doc_id: the ID of the delegated operation
+            label: the label to set
+
+        Returns:
+            a :class:`fiftyone.factory.repos.DelegatedOperationDocument`
+        """
+        return self._repo.set_label(_id=doc_id, label=label)
```

Comment:
The repo `set_label` method are positional arguments, correct? Should it be
```suggestion
        return self._repo.set_label(_id, label)
```